.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!--
+Allo' allo'! 
+Thanks for taking the time to submit a pull request ❤
+
+Please make sure you read and fulfill our pull request guidelines:
+https://github.com/yeoman/yeoman/blob/master/contributing.md
```

Comment:
```suggestion
https://github.com/yeoman/.gitthub/blob/master/.github/contributing.md
```

Reply:
```suggestion
https://github.com/yeoman/.github/blob/master/.github/contributing.md
```

Reply:
Great catch on the `gitthub` one :monocle_face:!